chromium_src/net/device_bound_sessions/session_binding_utils.cc

```diff
@@ -0,0 +1,15 @@
+/* Copyright (c) 2024 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "net/device_bound_sessions/session_binding_utils.h"
+
+#define RSA_PKCS1_SHA1 \
+  ECDSA_SHA384:        \
```

Comment:
wasn't there some previous override for this? Where did that one go?

Reply:
Yes, they copied the function from `components/signin/public/base/session_binding_utils.cc`.